src/DataAccess/DoctrineDonationRepository.php

```diff
@@ -31,15 +33,17 @@ public function __construct( EntityManager $entityManager ) {
 	}
 
 	public function storeDonation( Donation $donation ) {
+		$doctrineDonation = $this->newDonationEntity( $donation );
+
 		try {
-			$this->entityManager->persist( $this->newDonationEntity( $donation ) );
+			$this->entityManager->persist($doctrineDonation  );
```

Comment:
![lostinspacelogo](https://cloud.githubusercontent.com/assets/146040/13284553/3f4e3aa8-daf5-11e5-974a-39628a178ada.png)

Seriously though, what happened with the CS rule that checked this...